app/controllers/decidim/anonymous_codes/admin/code_groups_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Decidim
+  module AnonymousCodes
+    module Admin
+      class CodeGroupsController < ApplicationController
+        helper_method :groups
+
+        def index; end
+
+        def new; end
+
+        def create; end
+
+        private
+
+        def groups
+          AnonymousCodes::Group.all
```

Comment:
```suggestion
          AnonymousCodes::Group.for(current_organization)
```
At the moment will be like this. Then check pagination

Reply:
Thanks. Done